### PR TITLE
Adds Romerol (the REAL zombie powder) to uplink

### PR DESCRIPTION
**Pull request by coiax**
:cl: coiax
add: Adds Romerol to the traitor uplink for 25 TC. (This means you need a discount, or to work with another traitor to afford it). Romerol is a highly experimental bioterror agent which silently create dormant nodules to be etched into the grey matter of the brain. On death, these nodules take control of the dead body, causing limited revivification, along with slurred speech, aggression, and the ability to infect others with this agent.
del: Zombie infections are no longer visible on MediHUD.
del: Zombies no longer tear open airlocks, since they can just smash them open just as fast.
del: Zombies are no longer TOXINLOVING.
/:cl:

### CODE REVIEW

File: code/modules/uplink/uplink_item.dm [hunk 1]
```diff
@@ -632,6 +632,13 @@ var/list/uplink_items = list() // Global list so we only initialize this once.
 	cost = 6
 	surplus = 50
 
+/datum/uplink_item/stealthy_weapons/romerol_kit
+	name = "Romerol"
+	desc = "A highly experimental bioterror agent which creates dormant nodules to be etched into the grey matter of the brain. On death, these nodules take control of the dead body, causing limited revivification, along with slurred speech, agression, and the ability to infect others with this agent."
```
Comment (swindly) [inline code/modules/uplink/uplink_item.dm]: aggression

**Comment by PKPenguin321** [comment]
Remember when fungal tb was added to the uplink like this and it was used every traitor round and derailed it every single time?

**Comment by Armhulen** [comment]
I'll be honest folks

that name is extremely boring

**Comment by ExcessiveUseOfCobblestone** [comment]
i'm assuming you fixed the bugs associated with zombies

**Comment by PKPenguin321** [comment]
If my last post wasn't clear enough: This is an awful idea

**Comment by XDTM** [comment]
Please, if you want to add it the last place i'd put it is _roundstart_

**Comment by coiax** [comment]
Ah, but it doesn't happen at roundstart, because the reagent only creates the potential for zombies. You still need to kill the person afterwards.

**Comment by ShadowDeath6** [comment]
>poisoning a decomposing body and expecting it to do something

**Comment by Kromgar** [comment]
Merge it without traitor access make it something traitors can tc trade for

**Comment by Robustin** [comment]
Keep in mind these are "new" zombies that can be slipped, stunned, and generally meme'd in every possible way now that dismemberment is a thing. Last I checked a zombie is like a naked person in clown shoes (slow) coming at you with an e-dagger.

I think this works for traitor rounds because its utterly ineffective unless someone is killing a lot of people, but if someone is killing a lot of people (aka murderboning), then bringing people back into the round for revenge is better than just observing for the rest of the round. 

I never had a zombie round that didn't have a significant (65%) approval rating after the fact. They are legit fun because they give the crew a threat that is far more manageable and interactive than the average traitor/DA round.

**Comment by PKPenguin321** [comment]
I guess it's maybe worth trying if zombies are now as weak as you say, but I can still see this being used every single round for a week and snowballing almost every time (especially since I'm sure some people will go out of their way to get zombied for free antag)

**Comment by XDTM** [comment]
I can get on board with zombies with weaknesses, but they seem like they should be a reward for doing one's job, like xenobio can release spiders. Maybe a zombie virus symptom?

**Comment by coiax** [comment]
@Kromgar aside from my overhaul of some zombie abilities, all of the code (including the reagent) is already present. The only thing this PR adds is a premade bottle of romerol.

File: code/modules/zombie/items.dm [hunk 2]
```diff
@@ -54,48 +51,11 @@
...
-		user.adjustCloneLoss(-hp_gained)
+		// zero as argument for no instant health update
+		user.adjustBruteLoss(-hp_gained, 0)
+		user.adjustToxLoss(-hp_gained, 0)
+		user.adjustFireLoss(-hp_gained, 0)
+		user.adjustCloneLoss(-hp_gained, 0)
+		user.updatehealth()
 		user.adjustBrainLoss(-hp_gained) // Zom Bee gibbers "BRAAAAISNSs!1!"
-
-/obj/item/zombie_hand/proc/tear_airlock(obj/machinery/door/airlock/A, mob/user)
```
Comment (Iamgoofball) [inline code/modules/zombie/items.dm]: uh

**Comment by optimumtact** [comment]
a lotta stealth zombie nerfs not declared in the pr body

**Comment by kevinz000** [comment]
no this doesn't need to be fungal tb: even worse edition.

**Comment by KorPhaeron** [comment]
@Xhuis 

Something like this would be good for job specific traitor items, provided it wasn't so easily bypassed.

That way you wouldn't see it every round, which is what I'm afraid of with this

**Comment by coiax** [comment]
@optimumtact only real nerf is the removal of airlock tearing, which is redundant when smashing the airlock with your force 15 claws is about as fast (I checked).

**Comment by FantasticFwoosh** [comment]
Get dental implant & pop before they execute you & dump your body in a locked room/space

Boom, kool aid your way outta there and bite down the nearest person you see for a traitor zombie tide & still get your objectives. Im calling that it'll last a week before it gets balanced-removed.

> 20tc is fair but its a end-game xenobiology toy (gold slime zombies), and you don't even need to ingest it yourself to harvest future larger supplies of romerol.

**Comment by coiax** [comment]
Quick point that gold slime zombies were removed long ago, when I made zombies mutant humans. I will remove the entry from the uplink, but keep the rest intact, so later possibly @Xhuis can add it to traitor chef or something.

**Comment by coiax** [comment]
@KorPhaeron no longer in uplinks, just some balance changes and overhauls.

**Comment by RemieRichards** [comment]
could just make it cost > 20 TC (so that traitors have to team up to buy it, say, 25 TC)

**Comment by KorPhaeron** [comment]
That's a pretty great idea Remie

**Comment by coiax** [comment]
Done.

**Comment by KorPhaeron** [comment]
If it's terrible we can always revert or make it traitor viro only

**Comment by Nabski89** [comment]
I just loaded this into a syringe gun as a virologist. 1 unit of blood with another deadly illness, 1 unit of this, 13 units of toxin/morphine (sue me I was running out of time). It was very deadly. Shuttle was called due to singularity. There was one greentext of steal the core and escape, and a 50% survival rate for the station.

**Comment by coiax** [comment]
Yeah Gene, you shot me. But I was wearing a full body radiation suit (honk) so the syringe bounced off.

**Comment by Nabski89** [comment]
I think I still spread it to a fair amount of people considering I only started a minute before the shuttle was called. I blame RNG for taking 30 units to get evolutionary accel as a symptom.

**Comment by coiax** [comment]
Oh no, there were definitely enough zombies to cause a panic, and I don't think the radiation storm helped the station much, with the radiation-proof zombies prowling around

**Pull request merged**